reana-ui/src/components/WorkflowActionsPopup.js

```diff
@@ -38,7 +38,7 @@ export default function WorkflowActionsPopup({ workflow, className }) {
   const isDeleted = status === "deleted";
   const isDeletedUsingWorkspace = isDeleted && size.raw > 0;
   const isRunning = status === "running";
-  const isSessionOpen = sessionStatus === "created";
+  const isSessionOpen = sessionStatus === "running";
```

Comment:
Works nicely :+1: 

BTW, now that we have a new design, we could amend the text _"Click on the Jupyter logo to access it"_ to say something like "Click on the Jupyter notebook badge to access it."

Reply:
Cosmetics: for the commit log, please use capital letter for Jupyter:

```
fix(ui): show Jupyter badge only after notebook is running (#416)
```